app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Project Information</h1>
+
+<h3>Project Name: <%= @project.name %></h3>
+<h3>Project Material: <%= @project.material %></h3>
+<h3>Challenge Theme: <%= @project.challenge.theme %></h3>
+<h3>Number of Contestants: <%= @project_count %>
```

Comment:
This is where you could instead call `contestant_count` on a @project to get the # of contestants rather than the # of projects